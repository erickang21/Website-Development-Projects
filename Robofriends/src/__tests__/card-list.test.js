import { shallow } from "enzyme";
import React from "react";
import CardList from "../CardList";

const mock = [
    {
        id:1,
        name:"terry",
        email:"limace@gmail.com"
    },
    {
        id:2,
        name:"terr",
        email:"limac@gmail.com"
    },
    {
        id:3,
        name:"ter",
        email:"lima@gmail.com"
    },
]

it("renders the card list with the right cards", () => {
    expect(shallow(<CardList robots={mock}/>)).toMatchSnapshot()
})
