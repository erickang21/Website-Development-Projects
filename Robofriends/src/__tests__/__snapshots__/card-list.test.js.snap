// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the card list with the right cards 1`] = `
Array [
  <Card
    email="limace@gmail.com"
    id={1}
    key="0"
    name="terry"
  />,
  <Card
    email="limac@gmail.com"
    id={2}
    key="1"
    name="terr"
  />,
  <Card
    email="lima@gmail.com"
    id={3}
    key="2"
    name="ter"
  />,
]
`;
